frontend/app/routes/_gcweb-app.personal-information.address.edit.tsx

```diff
@@ -66,6 +66,7 @@ export default function ChangeAddress() {
   const actionData = useActionData<typeof action>();
   const { userInfo } = useLoaderData<typeof loader>();
   const { t } = useTranslation(i18nNamespaces);
+  console.log(actionData);
```

Comment:
This should be removed.

Reply:
removed in commit https://github.com/DTS-STN/canadian-dental-care-plan/pull/214/commits/75ab629978d6061c4f6a3c49f2872e325be338af